copilot/fastapi/tvp_header.md

```
@@ -0,0 +1,268 @@
+---
+
+# Thinnest Viable Platform (TVP) about Leverage
+
+---
+
+![This Lego experiment shows our brains prefer adding  Here's why it matters](https://github.com/user-attachments/assets/ea8b4c7f-bacc-4041-871b-8c1f85916e0c)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)